src/components/widgets/registration/FormComponent.tsx

```diff
@@ -34,9 +36,14 @@ export const FormComponentBase = ({ className }: RegFormBaseProps) => {
     const [showFirework, setShowFirework] = useState(false);
     const { isMobile } = useScreenDetector();
 
+    const { mutate: sendFormData } = useCreateAppointments();
     const onSubmit = async (data: FieldValues) => {
```

Comment:
If you want to avoid hook changes it is possible to do

```suggestion
    const onSubmit = async (data: FormFields) => {
```
now getValues return default type because you did not passed any type to useFormContext. But it is available to do, so your type are gonna work perfectly


```javascript
    } = useFormContext<FormFields>();
```